setup.go

```diff
@@ -38,10 +38,10 @@ func setup(c *caddy.Controller) error {
 
 	nl := netlink.Handle{}
 
-	if exists, err := EnsureDummyDevice(nl, cfg.ifName, cfg.localIPs); err != nil {
+	if exists, err := EnsureDummyDevice(&nl, cfg.ifName, cfg.localIPs); err != nil {
```

Comment:
the function sig seems different, can you check this https://github.com/contentful-labs/coredns-nodecache/pull/7/files#diff-cbffad7e35cf353e10b64e9059bbe37aL15

Reply:
its an interface... I forgot to pass one parameter, should be good now